### PR TITLE
Update audio-conferencing-common-questions.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Adding an additional technical detail

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/audio-conferencing-common-questions.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The maximum length of time depends on who is in the meeting and the type of auth
 |:-----|:-----|
 |There are users who have joined using the Skype for Business or Microsoft Teams app or have dialed in to the meeting.  <br/> |The meeting ends if there are no changes to the attendee list after 24 hours.  <br/> |
 |All of the users are dialed in to the meeting but someone has used a PIN to enter the meeting.  <br/> |The meeting ends after 24 hours.  <br/> |
-|All of the users are dialed in to the meeting but there wasn't anyone who used a PIN to enter the meeting.  <br/> |The meeting ends after 4 hours.  <br/> |
+|All of the users are dialed in to the meeting but there wasn't anyone who used a PIN to enter the meeting.  <br/> | If anonymous users are allowed to start meetings, the meeting ends after 4 hours. If anonymous users aren't allowed to start meetings and all authenticated participants have left the meeting, the meeting ends 90 minutes after the last authenticated participant left. <br/> |
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-common-questions.md]:
```suggestion
|All of the users are dialed in to the meeting but there wasn't anyone who used a PIN to enter the meeting.  <br/> | If anonymous users are allowed to start meetings, the meeting ends after four hours. If anonymous users aren't allowed to start meetings and all authenticated participants have left the meeting, the meeting ends 90 minutes after the last authenticated participant left. <br/> |
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
I accepted a couple minor copy edits to test a bug fix implemented by GitHub. @tonysmit still needs to review and merge.

**Comment by CarolynRowe** [comment]
I was checking voice and ac PRs.  I'm going to go ahead and merge this one because it's from Oscar.

**Pull request merged**